programs/fileio.c

```diff
@@ -1864,34 +1866,36 @@ FIO_zstdErrorHelp(const FIO_prefs_t* const prefs,
         unsigned long long const windowSize = header.windowSize;
         unsigned const windowLog = FIO_highbit64(windowSize) + ((windowSize & (windowSize - 1)) != 0);
         assert(prefs->memLimit > 0);
-        DISPLAYLEVEL(1, "%s : Window size larger than maximum : %llu > %u\n",
+        DISPLAYLEVEL(1, "%s : Window size larger than maximum : %llu > %u \n",
```

Comment:
Why put a space here?

Reply:
It helps for the specific case where
the screen is already filled with characters on the same line,
whatever the reason (garbage, or other running process for example).
The final space character will help determine that this is the end of the line, 
and more importantly in this case the end of the numerical field.

It has helped me a few times in the past, so I tend to make it a habit nowadays.